.env.gitpod

```diff
@@ -27,7 +27,7 @@ MAXMIND_USER_ID = 123456
 MAXMIND_LICENCE_KEY = "xxxxxxxxxxxxxxxx"
 
 MICROSOFT_PROVIDER_ID = 1234
-MICROSOFT_LOCAL_FILE_DIRECTORY = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
+MICROSOFT_REMOTE_API_ENDPOINT = "https://tgwf-web-app-test.s3.nl-ams.scw.cloud/data-imports/ms-azure-ip-ranges-2022-04-25.json"
```

Comment:
The endpoint seems to specify a date at the end. My code is reliant on that date as it extracts it and calculates the days between today and last updated. Is there a reason it is a literal?